osu.Game/Screens/OnlinePlay/Matchmaking/Match/BeatmapSelect/BeatmapSelectGrid.cs

```diff
@@ -81,8 +79,6 @@ private void load(AudioManager audio)
             for (int i = 0; i < spinSamples.Length; i++)
                 spinSamples[i] = audio.Samples.Get($@"Multiplayer/Matchmaking/Selection/roulette-{i}");
 
-            randomRevealSample = audio.Samples.Get(@"Multiplayer/Matchmaking/Selection/random-reveal");
```

Comment:
@nekodex Note that this PR removes your random reveal sound. See file `MatchmakingSelectPanelRandom.cs` where the random revel animation resides.